middleware.go

```diff
@@ -45,7 +45,10 @@ func (m *middleware) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	defer func() {
 		if recovered := recover(); recovered != nil {
-			logjamRequest.Log(FATAL, fmt.Sprintf("%#v", recovered))
+			msg := fmt.Sprintf("%#v", recovered)
+			agent.Logger.Println(msg)
```

Comment:
Is this really a logjam error that we should log?

Reply:
It's a user error. I think it's helpful to send the info to logjam and that the program does not silently die. Whether requests actually makes it to logjam  is a different question. I'm really missing an at_exit handler concept.

Reply:
And at the very least it is useful for local development. We might even want to add a stacktrace here. What do you thing?

Reply:
Never mind the last point, we can think about that later.